presentation-api/controlling-ua/startNewPresentation_success-manual.html

```diff
@@ -39,11 +72,11 @@
 
                         // Check the instance of the connection
                         assert_true(connection instanceof PresentationConnection, 'The connection is an instance of PresentationConnection.');
-                    });
+                    }), function(e) { reject(e); });
+            });
```

Comment:
Thanks! It looks almost good to me. My concern is simply that we must wait for timeout even if the event is fired in incorrect order. I will PR which adds checking the event order.